website/src/docs/hotchocolate/v12/defining-a-schema/subscriptions.md

```diff
@@ -252,6 +252,8 @@ public async Book PublishBook(Book book, [Service] ITopicEventSender sender)
 
 If more complex topics are required, we can use the `ITopicEventReceiver`.
 
+> ⚠️ Note: Pay attention to the use of `SubscribeAndResolve` instead of `Subscribe`. As can be seen in the code-first example above, all subscription fields are in fact made of two methods (resolver and subscriber). When using `Subscribe`, the attributed method represents resolver method (with subscriber method generated at runtime) whereas `SubscribeAndResolve` denotes subscriber method (with resolver method generated at runtime).
```

Comment:
This is a bit unclear... I would recommend people not to use SubscribeAndResolve... we are deprecating it. Instead I would use the two separate methods. Subscribe and resolve leads to issues for the standard user with captured contexts.

Reply:
I think the intend for the note is to dissuade people from using SubscribeAndResolve... this should be clearer.

Reply:
In the V12 documentation it should be still available ... but for 13 we can remove it

Reply:
> We recommend not use `SubscribeAndResolve` as it very easily can lead to unexpected behavior with the state that is aggregated on not released.

For 12 we could have a very strong first sentence and for 13 remove it from the docs.

@tobias-tengler what do you think?

Reply:
I think we can remove it from the v13 documentation. We've already documented that it's deprecated in the upgrade guide so I think we should stop documenting it as well.

Reply:
@sunghwan2789 can you apply the changes so we can merge?

Reply:
I applied the changes. Thanks!